README.md

```diff
@@ -43,43 +42,35 @@ less complex and more composible.
 
 Install the released version of `sass` from CRAN:
 
-``` r
-install.packages("sass")
-```
+    install.packages("sass")
 
 Install the latest development build from GitHub:
 
-``` r
-# install.packages("remotes")
-remotes::install_github("rstudio/sass")
-```
+    # install.packages("remotes")
+    remotes::install_github("rstudio/sass")
 
 ### Basic usage
 
 To compile Sass into CSS, provide Sass to the `input` argument of the
 `sass()` function. `input` can be any of the following:
 
-  - An R string (as in the example below).
-  - A named `list()` defining [Sass
+-   An R string (as in the example below).
+-   A named `list()` defining [Sass
     variables](https://rstudio.github.io/sass/articles/sass.html#variables).
-  - A `sass_file()`, `sass_import()`, or `sass_layer()`.
-  - A nested `list()` comprising of all the above.
-
-<!-- end list -->
-
-``` r
-library(sass)
-sass(input = "
-  $size: 50%;
-  foo { margin: $size * .33; }
-")
-```
-
-``` css
-foo {
-  margin: 16.5%;
-}
-```
+-   A `sass_file()`, `sass_import()`, or `sass_layer()`.
+-   A nested `list()` comprising of all the above.
+
+<!-- -->
+
+    library(sass)
+    sass(input = "
+      $size: 50%;
+      foo { margin: $size * .33; }
+    ")
+
+    foo {
+      margin: 16.5%;
+    }
```

Comment:
Whoa, is this due to a change in rmarkdown?

Reply:
Yes. Seems like it. Or pandoc?

Reply:
Oh no. Not good

Reply:
I just manually fixed it for now (I think it's some (hopefully) temporary pandoc weirdness)